R/compile_report.R

```diff
@@ -76,6 +76,12 @@ compile_report <- function(file, quiet = FALSE, factory = getwd(),
   ## e.g. `foo_date/compiled_[date]_[time]/`, and will include `params` if
   ## provided (see point 3).
   
+  
+  # This is used for logging later in the function 
+  log_entry <- as.list(environment())
+  log_entry$dots <- list(...)
+  log_entry$timestamp <- Sys.time()
```

Comment:
Just checking: are we indeed just using a single timestamp for:

* the report output to generate `compiled_[params]_[timestamp]/`
* the log

If it is already the case fine but bet store it somewhere upstream.